package.json

```diff
@@ -142,6 +142,15 @@
         "title": "Login into Cluster with credentials",
         "category": "OpenShift"
       },
+      {
+        "command": "openshift.explorer.switchContext",
+        "title": "Manage Contexts",
```

Comment:
It is not really about managing contexts, but rather about changing contexts.

Reply:
There is a very funny thing here, the order in which the titles appear on the explorer bar is based on the title name. So to make the order correct I used something which starts after 'L' and before 'R' and hence Manage. Is there a way to sort the order ?

Reply:
The same way  it is done in context menus I guess.

Reply:
No, it doesn't take group ordering. It just sorts name alphabetically.
And the contents displayed are only there if it's under `"group": "navigation"`.

Can we rename it to `Manage Context Switch` ?

Reply:
okay, i think I have a solution. sending the updates.

Reply:
Fixed by https://github.com/redhat-developer/vscode-openshift-tools/pull/1019/commits/886eb7f2b9b0df729efcf33a9e36466004005438